06-hot-reloading-with-directives/services/reviews/schema.js

```diff
@@ -0,0 +1,68 @@
+const { makeExecutableSchema } = require('@graphql-tools/schema');
+const { stitchingDirectives } = require('@graphql-tools/stitching-directives');
+
+const NotFoundError = require('../../lib/not_found_error');
+const readFileSync = require('../../lib/read_file_sync');
+
+const { stitchingDirectivesTypeDefs, stitchingDirectivesValidator } = stitchingDirectives();
+
+const typeDefs = `
+  ${stitchingDirectivesTypeDefs}
+  ${readFileSync(__dirname, 'schema.graphql')}
+`;
+
+// data fixtures
+const usernames = [
```

Comment:
This secondary cache of user data is extremely atypical from common application design, therefore doesn't really feel appropriate here to me. If the intent is to illustrate fields shared between services, that could be its own extremely small and simple demo. For simplicity with everything else going on, `usernames` feels like it should just merge from the accounts service, which would mirror the most typical database design.

Reply:
Could be, just copied from the Federation demo....

Reply:
Ah, yeah, that demo sucks. It’s trying so hard to show off federation that it builds something contrived and, ironically, makes federation less intuitive given that you’re trying to discern haphazard application design from the techniques in federation.